.pre-commit-config.yaml

```diff
@@ -13,4 +13,4 @@ repos:
     hooks:
     -   id: fmt
     -   id: clippy
-        args: ["--", "-D", "warnings", "-W", "clippy::pedantic"]
+        args: ["--all-targets", "--", "-D", "warnings", "-W", "clippy::pedantic"]
```

Comment:
Can you add a comment that clippy:pedantic won't apply to tests/examples unless you include `--all-targets`